### PR TITLE
Various changes

### PR DESCRIPTION
**Pull request by IllusionMan1212**
- `Help` command
- `explain list` command and `explain delete` command
and more stuff that i can't think of rn

image
image

### CODE REVIEW

File: Vita3KBot/Commands/Help.cs [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            if (Context.Channel.GetType().Name != "SocketDMChannel") {
+                var user = Context.User as IGuildUser;
+                if (user.GuildPermissions.Administrator)
+                    isAdmin = true;
+            }
+
+            EmbedBuilder helpEmbed = new EmbedBuilder();
+            helpEmbed.WithTitle("Help");
+            helpEmbed.WithDescription("These are all the available commands, specify a command for more information about it.");
+            helpEmbed.WithColor(Color.Orange);
```
Comment (1whatleytay) [inline Vita3KBot/Commands/Help.cs]: Chain if you feel like it!

File: Vita3KBot/Commands/Help.cs [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            await ReplyAsync(embed: helpEmbed.Build());
+        }
+
+        [Command]
+        public async Task Help() {
+            List<string> commandList = new List<string>();
+            string commands = "";
+
+            bool isAdmin = false;
+            if (Context.Channel.GetType().Name != "SocketDMChannel") {
```
Comment (1whatleytay) [inline Vita3KBot/Commands/Help.cs]: Same `if (!(Context.Channel is IPrivateChannel)) { ... }` here?

**Comment by IllusionMan1212** [comment]
Changes done

**Comment by 1whatleytay** [comment]
Awesome. Merging and pushing to the bot.

**Pull request merged**